tests/framework/results/test_stratification.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from vivarium.framework.results.stratification import Stratification
+
+##########################
+# Mock data and fixtures #
+##########################
+NAME = "hogwarts_house"
```

Comment:
I've never seen the movies :( 

(though I have read a few of the first books!)